static/app/views/starfish/views/webServiceView/starfishView.tsx

```diff
@@ -123,12 +124,13 @@ export function StarfishView(props: BaseStarfishViewProps) {
                     bottom: '0',
                   }}
                   aggregateOutputFormat="rate"
+                  rateUnit={RateUnits.PER_SECOND}
```

Comment:
l: it feels weird to me that we have to duplicate the `rateUnit` both here and within in `valueFormatter`. Something to improve for later on maybe.

Reply:
Yep, for sure! I think the unit should be part of the `Series[]` to be honest, but I haven't looked into it deeply yet